File: app/src/main/java/com/kml/data/internalRoomDatabase/GameDao.java
package com.kml.data.internalRoomDatabase;

import androidx.lifecycle.LiveData;
import androidx.room.Dao;
import androidx.room.Query;
import com.kml.data.models.Game;
import java.util.List;

@Dao
public interface GameDao
{
    @Query("SELECT * FROM gameTable")
    LiveData<List<Game>> getAllGames();
}

File: app/src/main/java/com/kml/data/internalRoomDatabase/GameDao.kt
package com.kml.data.internalRoomDatabase

import androidx.lifecycle.LiveData
import androidx.room.Dao
import androidx.room.Query
import com.kml.models.Game

@Dao
interface GameDao {
    @get:Query("SELECT * FROM gameTable")
    val allGames: LiveData<List<Game>>
}